### ISSUE TITLE
Spyder unresponsive in WinPython-64bit-3.6.2.0Qt5 on Windows 7

### ISSUE THREAD

**Issue by jondo** [winpython/winpython#566]
I am on Windows 7 Professional 64bit.
In WinPython-64bit-3.6.2.0Qt5, when I launch Spyder.exe, only an unresponsive frame of the Spyder window becomes visible. There are 6 instances of QtWebEngineProcess.exe running, but in contrast to #427 none of them uses much CPU. Also, killing the QtWebEngineProcess.exe processes does not change the situation this time, and #427's QWebEngineTest.py shows no errors.

During launch the Spyder window renders correctly, then a console window with the title `C:\Program Files\WinPython-64bit-3.6.2.0Qt5\tools\pandoc.exe` opens for two seconds, and when this closes again, Spyder has already stopped to redraw its frame.

In Process Explorer I see the following process tree:
![3 6 2 0-processes](https://user-images.githubusercontent.com/21648/30487775-bb06fa40-9a34-11e7-934d-cab18b485a91.png)

Why is there `jupyter-notebook.exe`involved?
In WinPython-64bit-3.6.1.0Qt5, Spyder works and shows the following processes:
![3 6 1 0-processes](https://user-images.githubusercontent.com/21648/30487672-7327b23c-9a34-11e7-8e33-b4575fd271a9.png)

**Comment by stonebig** [comment]
This is an issue between Qt5.8 and Windows 7. I gave up on Windows 7 maintenance. 

Windows 7 users should use an old WinPython version of pre-Windows 10 initial release time, or a version with Qt5.6 at most.

**Comment by Patrick-Cole** [comment]
Out of interest, I use windows 7 64 bit, and the 3.5.4.0Qt5 release - and everything works perfectly. I suspect (based on a bug I saw, can't remember what) that this may be a problem with the compatibility of the latest Spyder, one of its dependencies and python 3.6. Going back to python 3.5 solved this for me.

**Comment by jondo** [comment]
OK, I'll have to trade Windows 7 against Python 3.6. @stonebig, feel free to close this as Wontfix.

**Comment by jondo** [comment]
Interestingly, I have now got it working.
For this, I restarted with the installer, but I kept the freshly extracted WinPython directory in my user profile instead of copying it to `%programfiles%` as admin before the first spyder launch.

In fact I tested it twice this way. The first time I got this dialog at the first launch, but Spyder (and the kernel) worked:
> Render Report Error
> ---------------------------
> Can only launch a kernel on a local interface. Make sure that the '*_address' attributes are configured properly. Currently valid addresses are: ['140.78.127.112', '192.168.56.1', '0.0.0.0', '']

No idea why it thought that I wanted a remote kernel.

The second time I did not work locally but connected via `xfreerdp` remote desktop. In that case, after extraction from installer Spyder worked without showing that error.
